.travis/script.sh

```diff
@@ -432,6 +432,13 @@ set -x
     COVERAGE=1 CFLAGS=-m32 CXXFLAGS=-m32 LDFLAGS=-m32 make -f examples/Makefile-posix check || die
 }
 
+[ $BUILD_TARGET != posix-app-cli ] || {
+    ./bootstrap || die
+    make -f examples/Makefile-posix || die
+    make -f src/posix/Makefile-posix || die
+    OT_CLI_PATH="$(pwd)/$(ls output/posix/*/bin/ot-cli)" RADIO_DEVICE="$(pwd)/$(ls output/*/bin/ot-ncp-radio)" COVERAGE=1 make -f src/posix/Makefile-posix check || die
```

Comment:
Not for this PR, but let's figure out how to run these tests faster than real-time in the future.